tests/test_other.py

```diff
@@ -10274,7 +10274,8 @@ def test_standalone_syscalls(self):
       for engine in WASM_ENGINES:
         self.assertContained(expected, run_js('test.wasm', engine))
 
-  @no_windows('TODO: fix setjmp.h on clang on windows on ci')
+  @requires_native_clang
+  @no_windows('compiles on windows but does not link https://github.com/emscripten-core/emscripten/pull/11382#pullrequestreview-428902638')
```

Comment:
Its this covered by requires_native_clang alone?

Reply:
I don't think so, since we currently report we have native clang on windows, don't we? (But we are discussing changing that to always assume "no" on windows, https://github.com/emscripten-core/emscripten/pull/11382/files#r438902352).

Reply:
Its seems that the linking problem is problem with the broken clang environment on the bots rather than a general windows-incompatibility.

The windows bots should signal they don't support native clang (at least native linking) because its not working yet.

If I am a normal windows user I might be sure have a good windows build setup I might want to still run this test.. or at least try.

On the other hand if you think the code itself is simply not windows compatilbe... then no_windows would be the correct thing to do.

IIUC, the test requires a working native clang setup and our windows CI bots don't have that, but that doesn't mean the test won't work if one does have such a thing locally.

Reply:
Sounds good, I can remove this part.

Before landing this PR we'll need to make the bots report "no native clang" on windows. If you aren't already doing that, I can do it in this PR, or open a new one?

Reply:
Added that line to this PR, and removed the line you asked.